tsconfig.json

```diff
@@ -1,16 +1,12 @@
 {
   "compilerOptions": {
     "lib": ["es2022"],
-    "moduleResolution": "NodeNext",
-    "module": "NodeNext",
+    "moduleResolution": "Bundler",
+    "module": "es2022",
```

Comment:
Are we not safe using esnext or nodenext here for `module`?

Reply:
See below / next comment.

Reply:
What about using "esnext" here instead of "es2022"?

Reply:
It can be used though it wouldn't make any difference other than never having to change it in the future; there is nothing presently done in `jsdoccomment` higher than `es2022`.

It's more of a personal preference thing.

Reply:
Let's go for `esnext` and then I think we're done.

Reply:
> Let's go for `esnext` and then I think we're done.

Done.